### PR TITLE
OCPBUGS#9210: Updated folder parameter info in vSphere UPI docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-9210](https://issues.redhat.com/browse/OCPBUGS-9210)

Version(s):

4.14 through to 4.10

Link to docs preview:
* [Installing a cluster on vSphere with user-provisioned infrastructure](https://62809--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere)
* [Installing a cluster on vSphere with user-provisioned infrastructure and network customizations](https://62809--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-vsphere-config-yaml_installing-vsphere-network-customizations)
* [Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure](https://62809--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-vsphere-config-yaml_installing-restricted-networks-vsphere)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-config-yaml.adoc [hunk 1]
```diff
@@ -154,7 +154,7 @@ value must match the number of control plane machines that you deploy.
...
-<10> Optional parameter For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster, omit this parameter.
+<10> Optional parameter For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster and you do not want to use the default `StorageClass`, named `thin`, you can omit the `folder` parameter from the `install-config.yaml` file.
```
Comment (JoeAldinger) [inline modules/installation-vsphere-config-yaml.adoc:157]:
```suggestion
<10> Optional parameter For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster and you do not want to use the default `StorageClass` object, named `thin`, you can omit the `folder` parameter from the `install-config.yaml` file.
```

Comment (JoeAldinger) [reply modules/installation-vsphere-config-yaml.adoc:157]: I looked around the repo and I _think_ `StorageClass` is an object.

**Review by JoeAldinger — COMMENTED** [review]
small suggestion

File: modules/installation-vsphere-config-yaml.adoc (hunk 1 above)
Comment (dfitzmau) [reply modules/installation-vsphere-config-yaml.adoc]: Thanks for checking, @JoeAldinger . I was not fully sure of what component this was as I seen "standard", "metadata", and so on. I kept it generalized for this reason. Now with a second opinion, you're right about "object". I updated the PR accordingly.

**Review by jldohmann — APPROVED** [review]
LGTM

**Comment by jldohmann** [comment]
@dfitzmau it looks like there were some merge conflicts with the 4.10, 4.11, and 4.12 branches. You'll need to perform manual cherrypicks for these changes and open a new PR against each branch. Please reach out if you have any questions, and ping me when the PRs are up and i'll be happy to merge. Thank you!

**Pull request merged**